packaging/os/yum.py

```diff
@@ -574,6 +600,8 @@ def install(module, items, repoq, yum_basecmd, conf_file, en_repos, dis_repos):
                 module.fail_json(**res)
 
             pkg_name = local_name(module, spec)
+            pkg_nevra = local_nvra(module, spec)
+
             # look for them in the rpmdb
             if is_installed(module, repoq, pkg_name, conf_file, en_repos=en_repos, dis_repos=dis_repos):
```

Comment:
ppersad, actually it was decided that the name is the approprate check for "install". https://github.com/ansible/ansible-modules-core/pull/3311

Having install check the nevra instead of name, would effectively rebreak the issue that 3311 fixed.

I'll explain more in an issue comment.